pyproject.toml

```diff
@@ -15,6 +15,7 @@ requests = "^2.32.3"
 requests-futures = "^1.0.1"
 flagsmith-flag-engine = "^5.1.0"
 sseclient-py = "^1.8.0"
+pydantic = "^2.8.2"
```

Comment:
I guess there's a slight concern here that we might end up with version conflicts between the client and the engine. Can we be even less restrictive here since the usage of pydantic is so minimal?

Reply:
Good catch, this is addressed.